schematic/models/metadata.py

```diff
@@ -273,10 +273,6 @@ def validateModelManifest(
 
             return errors, warnings
 
-        # check if suite has been created. If so, delete it
-        if os.path.exists("great_expectations/expectations/Manifest_test_suite.json"):
-            os.remove("great_expectations/expectations/Manifest_test_suite.json")
-
```

Comment:
No longer cleaning up the test suite run here. Instead relying on the cleanup to happen in the `finally` block after the expectation runs.